conf/stub.config

```diff
@@ -1,14 +1,133 @@
-params {
-    outdir= "${baseDir}/results_stub"
-    reads= "${baseDir}/data/mock_data/*_{R1,R2}*.fastq.gz"
-    reference_fasta= "${baseDir}/data/mock_data/NZ_CP013742.1.fasta"
-    reference_gbk= "${baseDir}/data/mock_data/NZ_CP013742.1_full.gb"
-}
-
 process {
+
+   withName:
+   'PICARD_CREATE_SEQUENCE_DICTIONARY' {
+       container = "gliderlabs/alpine"
+       cpus = 1
+       memory = "2 GB"
+   }
+
+
+   withName:
+   'BWA_INDEX' {
+       container = "gliderlabs/alpine"
+       cpus = 1
+       memory = "2 GB"
+   }
+
+   withName:
+   'BWA_MEM' {
+       container = "gliderlabs/alpine"
+       cpus = 1
+       memory = "2 GB"
+   }
+
+   withName:
+   'SAMTOOLS_FAIDX' {
+       container = "gliderlabs/alpine"
+       cpus = 1
+       memory = "2 GB"
+   }
+
+   withName:
+   'SAMTOOLS_INDEX' {
+       container = "gliderlabs/alpine"
+       cpus = 1
+       memory = "2 GB"
+   }
+
+   withName:
+   'SAMTOOLS_SORT' {
+       container = "gliderlabs/alpine"
+       cpus = 1
+       memory = "2 GB"
+   }
+
+
+   withName:
+   'GATK_HAPLOTYPE_CALLER' {
+       container = "gliderlabs/alpine"
+       cpus = 1
+       memory = "2 GB"
+   }
+
+
+   withName:
+   'GATK_MARK_DUPLICATES_SPARK' {
+       container = "gliderlabs/alpine"
+       cpus = 1
+       memory = "2 GB"
+   }
+
+   withName:
+   'FASTANI' {
+       container = "gliderlabs/alpine"
+       cpus = 1
+       memory = "2 GB"
+   }
+
+    withName:
+    'FASTQC' {
+        container = "gliderlabs/alpine"
+        cpus = 1
+        memory = "2 GB"
+    }
+
     withName:
-    ".*" {
-        cpus = 2
```

Comment:
@Mxrcon , perhaps there has been a misunderstanding. We don't need to specify the process names at all. We simply needed to add a `process.container = "ubuntu:20.04"` to the previous `stub.config`.